py/test/selenium/webdriver/common/bidi_network_tests.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import pytest
+
+from selenium.webdriver.common.bidi.network import UrlPatternString
+
+
+@pytest.mark.xfail_firefox
+@pytest.mark.xfail_safari
+@pytest.mark.xfail_edge
```

Comment:
Is this not working against Firefox and Edge?

Reply:
It was working with edge. I've pushed a commit to remove `xfail_edge`.

As for firefox, it fails with "unsupported operation" at `network.continueRequest`. I see mentions of `network.continueRequest` at https://wiki.mozilla.org/Firefox/Meeting/9-July-2024 , but I recall experiencing a similar problem with firefox when testing #14345.